### ISSUE TITLE
[BUG] Beast Traning not working with German language selected

### ISSUE THREAD

**Issue by mongsns** [Karl-HeinzSchneider/WoW-DragonflightUI#290]
The hunter beast training window is not working when german language is selected (with english selected, everythin works fine)

Old style and no train button:
![grafik](https://github.com/user-attachments/assets/70345184-c41b-417c-aeb7-a5b34f558257)

 - WoW: Classic (1.15.5.57979)
 - Addon v0.19.11

**Comment by Karl-HeinzSchneider** [comment]
hey,
are you getting any lua errors? My SoD hunter is on my 2nd acc, which is without subscription at the moment. So I need to level a new hunter to check.

**Comment by mongsns** [comment]
Yes I get a lua error: Here is the bugsack output:

```
4x DragonflightUI/Mixin/ProfessionTwo.mixin.lua:916: attempt to index local 'profData' (a nil value)
[string "@DragonflightUI/Mixin/ProfessionTwo.mixin.lua"]:916: in function `GetProfessionID'
[string "@DragonflightUI/Mixin/ProfessionTwo.mixin.lua"]:865: in function `UpdateHeader'
[string "@DragonflightUI/Mixin/ProfessionTwo.mixin.lua"]:118: in function `Refresh'
[string "@DragonflightUI/Mixin/ProfessionTwo.mixin.lua"]:186: in function `Minimize'
[string "@DragonflightUI/Mixin/ProfessionTwo.mixin.lua"]:64: in function <DragonflightUI/Mixin/ProfessionTwo.mixin.lua:53>
[string "=[C]"]: in function `Show'
[string "@Blizzard_UIParentPanelManager/Classic/UIParentPanelManager_Shared.lua"]:603: in function `SetUIPanel'
[string "@Blizzard_UIParentPanelManager/Classic/UIParentPanelManager_Shared.lua"]:448: in function `ShowUIPanel'
[string "@Blizzard_UIParentPanelManager/Classic/UIParentPanelManager_Shared.lua"]:316: in function <...PanelManager/Classic/UIParentPanelManager_Shared.lua:312>
[string "=[C]"]: in function `SetAttribute'
[string "@Blizzard_UIParentPanelManager/Classic/UIParentPanelManager_Shared.lua"]:1091: in function `ShowUIPanel'
[string "@Blizzard_UIParent/Vanilla/UIParent.lua"]:1005: in function <Blizzard_UIParent/Vanilla/UIParent.lua:494>

Locals:
self = <table> {
}
nameLoc = "Bestienausbildung"
rank = 0
maxRank = 0
skillID = nil
profData = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = "attempt to index local 'profData' (a nil value)"
professionDataTable = <table> {
 197 = <table> {
 }
 773 = <table> {
 }
 356 = <table> {
 }
 186 = <table> {
 }
 666 = <table> {
 }
 667 = <table> {
 }
 202 = <table> {
 }
 129 = <table> {
 }
 164 = <table> {
 }
 165 = <table> {
 }
 182 = <table> {
 }
 171 = <table> {
 }
 794 = <table> {
 }
 185 = <table> {
 }
 393 = <table> {
 }
 755 = <table> {
 }
 333 = <table> {
 }
}
```

**Comment by Karl-HeinzSchneider** [comment]
The blizzard functions for the two "craft" professions - enchanting, and wierdly enough beast training - are kind of buggy; I think I found a fix with: 780593c95dee3d233a3afde2aef7176366c57996
Maybe I can test more if my hunter gets to 10.. I'm still at 8, and its not that fun atm.
